bin/migrate.sh

```diff
@@ -0,0 +1,3 @@
+!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

![image](https://github.com/user-attachments/assets/f77600cc-b647-4a38-ba51-9655b05fbc01)